src/ape/managers/project/manager.py

```diff
@@ -447,16 +447,24 @@ def _get_contract(self, name: str) -> Optional[ContractContainer]:
 
         return None
 
-    # @property
-    # def meta(self) -> PackageMeta:
-    #     return PackageMeta(**self.config_manager.get_config("ethpm").serialize())
+    @property
+    def meta(self) -> PackageMeta:
+        """
+        Populate package manifest with metadata as per EIP
+        https://eips.ethereum.org/EIPS/eip-2678#the-package-meta-object
+        """
+        meta = self.config_manager.get_config("meta")
+        try:
+            return PackageMeta(**meta.serialize())
+        except Exception as e:
+            raise ConfigError(f"Incorrect configuration of package metadata:\n{meta}") from e
```

Comment:
this code isn't getting ran when compiling or extracting manifest. I also have the following error in mypy `src/ape/managers/project/manager.py:459: error: "PluginConfig" has no attribute "serialize"`

Reply:
it does get ran when I do this command in console: `In [1]: project.Token.project_manager.meta`